### PR TITLE
Rachel Lab 1 & 2 GTG

### PR DESCRIPTION
**Pull request by junedeor**
Lab 3 ready for grading

### CODE REVIEW

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 1: Bio](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/01%20Bio.md)
**Needs Work**
* Comments:
    * Awesome profile, cats are the best!
    * Image file-paths need to be relative file paths to be seen on other computers:
        * Edit file-path for image `Young_Salem.jpeg` so it can be rendered on any computer.

[Lab 2: Company Home](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/02%20Company.md)
**Completed**
* Comments:
    * Responsive webpage.
    * Great functionality where images scale with container size rather than hard-coding image sizes.

File: code/rachel/HTML/static/lab03_style.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        flex-wrap: wrap;
+        justify-content: flex-start;
+    }
+    h3 {
+        padding-right: 1em;
+    }
+    div p {
+        display: flex;
+        justify-content: flex-start;
+    }
```
Comment (brucestull) [inline code/rachel/HTML/static/lab03_style.css:106]: The `@media (min-width: 1000px)` media query needs a closing `}`.

**Comment by junedeor** [comment]
Lab 03 correction was made and it's ready for review

**Review by brucestull — APPROVED** [review]
[Lab 1: Bio](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/01%20Bio.md)
**Completed**

[Lab 3: Blog](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/03%20Blog.md)
**Completed**
* Comments:
    * Lab passes.
    * One thing you could or should change:
        * You have links with an href `href="dfsdf"`:
            * This breaks the page, a better way it so use this since it doesn't route to non-existent page:
                * `href="#"`
        * Try it out and see what happens.

**Pull request merged**